cmd/webhook/main.go

```diff
@@ -42,6 +44,12 @@ var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	v1alpha1.SchemeGroupVersion.WithKind("TriggerBinding"):        &v1alpha1.TriggerBinding{},
 	v1alpha1.SchemeGroupVersion.WithKind("TriggerTemplate"):       &v1alpha1.TriggerTemplate{},
 	v1alpha1.SchemeGroupVersion.WithKind("Trigger"):               &v1alpha1.Trigger{},
+
+	v1beta1.SchemeGroupVersion.WithKind("ClusterTriggerBinding"): &v1beta1.ClusterTriggerBinding{},
+	v1beta1.SchemeGroupVersion.WithKind("EventListener"):         &v1beta1.EventListener{},
+	v1beta1.SchemeGroupVersion.WithKind("TriggerBinding"):        &v1beta1.TriggerBinding{},
+	v1beta1.SchemeGroupVersion.WithKind("TriggerTemplate"):       &v1beta1.TriggerTemplate{},
+	v1beta1.SchemeGroupVersion.WithKind("Trigger"):               &v1beta1.Trigger{},
```

Comment:
Based on the above changes i assume we are not adding `ClusterInterceptor` to beta right ?

Reply:
Yes, they will be in the v1alpha1 type but can be used from v1beta1 triggers